content/01-definitions/definitions.md

```diff
@@ -0,0 +1,24 @@
+
+**Artificial intelligence** (AI) is broadly defined as an effort to automate intellectual tasks normally performed by human beings.2 AI includes machine learning and deep learning as well as approaches that do not involve any learning. Symbolic AI, for example, is premised on the ability of programmers to hard code a sufficiently large number of rules for manipulating knowledge (this is also known as *expert systems*). Symbolic AI can serve well for logical, well-defined problems.  
+
+**Machine learning** (ML) is better suited for more complex and intractable problems, such as image classification, speech recognition, language translation and probabilistic anomaly detection. Machine learning is a subset of AI where data can be used to train computers to detect patterns and mimic human decision-making. A machine-learning system is trained rather than explicitly programmed. It’s presented with many examples relevant to a task, and it finds statistical structure in these examples that eventually allows the system to come up with rules for automating the task. Machine learning involves using algorithms to detect patterns in data and then use those learned patterns to predict an outcome on new data. Generally, this involves an iterative process and allows the machine learning system to determine predictions without being explicitly programmed.  
+
+**Deep learning** is a subset of ML that can process a wider range of data resources including text, images and audio clips and typically requires less data preprocessing by humans. The “deep” part refers to learning successive layers of increasingly meaningful representations all learned from training data. The depth refers to the number of layers that contribute to a model of the data.  In contrast, other approaches to machine learning tend to focus on learning only one or two layers of representations of the data; hence, they’re sometimes called shallow learning. In common-speak, the application of neural networks models refers to deep learning.  
+
+**Natural Language Processing** (NLP) heavily leverages Machine Learning as well as Deep Learning to allow computers to interact with humans in their own natural language whether written or oral. NLP applications attempt to understand natural human communication, either written or spoken, and communicate in return with us using similar, natural language.  
+
+**Robotic Process Automation** (RPA) is a type of automation that mimics the activity of a human when they carry out a structured, repetitive task or process, little judgement. It is generally rule based, can’t learn outside parameters, so not considered AI; however, RPA can be built with learning modules on complex tasks.  
+
+Figure 1 diagram below depicts the AI ecosystem and the relationship between AI, ML, Deep Learning, NLP and RPA. 
+
+![Figure 1: The AI ecosystem](../../static/images/ai-ecosystem.png)
```

Comment:
There are two ways to link images. I prefer the first method:

1. Upload the image into the git repository in a path that matches the name of the file, e.g. `static/01-definitions/definitions/ai-ecosystem.png`. Link using the file name (the suggestion offered in this comment).
2. Upload the file anywhere in the `static/` directory. All links must be hard-coded with paths, e.g. `/ooi-toolkit-ds/images/ai-ecosystem.png`.

```suggestion
![Figure 1: The AI ecosystem](ai-ecosystem.png)
```

Reply:
Thanks. I uploaded image to the path `static/images/01-definitions/definitions/ai-ecosystem.png`

Reply:
Should I create a separate pull request?

Reply:
Looks like I gave you bad advice about where to store the files. I'll push a new commit to the Pull Request with the fix and explain it in a new comment.